src/protocol/binary_serialization.h

```diff
@@ -19,15 +20,11 @@ namespace OpcUa
   template<class Stream, class Container>
   inline void SerializeContainer(Stream& out, const Container& c, uint32_t emptySizeValue = ~uint32_t())
   {
-    if (c.size() == 0)
-    {
+    if (c.empty()) {
```

Comment:
Hello. Thank you for your pull request!
Could you please format {} to place them after 'if' 'else'. This style  used in project.